### ISSUE TITLE
Missing rows in the dataframe; not showing up on the map

### ISSUE THREAD

**Issue by perfectly-preserved-pie** [perfectly-preserved-pie/larentals#24]
Some of the rows aren't getting rendered on the map. Why?

`INFO:root:The original dataframe has 1296 rows. There are 1208 rows in the filtered dataframe. There are 1208 markers on the map.`

That's ~90 rows missing, even with all options/checked. There shouldn't be any actual filtering going on.

**Comment by perfectly-preserved-pie** [comment]
Using https://github.com/perfectly-preserved-pie/larentals/commit/e34227018251786f4f6aec8b1e40e719b7182a3b I was able to generated a CSV containing the missing rows.

The only thing _every_ missing row has in common is a malformed `popup_html` cell. Why?

**Comment by perfectly-preserved-pie** [comment]
(maybe related) I'm missing some subtypes and rental terms:

```
>>> df.subtype.unique()
array(['CONDO', 'APT', 'SFR/D', 'SFR/A', 'STUD/A', 'SFR', 'APT/A',
       'MANL/D', 'TPLX/A', 'QUAD/A', 'CONDO/A', 'APT/D', 'DPLX/A',
       'TPLX/D', 'TWNHS/A', 'TWNHS', nan, 'DPLX/D', 'CONDO/D', 'STUD/D',
       'QUAD/D', 'TWNHS/D', 'RMRT/A', 'CABIN/D', 'RMRT/D'], dtype=object)
```

```
>>> df.Terms.unique()
<StringArray>
['12M', 'NG', <NA>, '6M', 'MO', '24M', 'STL']
Length: 7, dtype: string
```

f6d1404381e01fb25d18912b3a9ed82ce800148e and 410703fbf4e8c51335a667b4ffd88af2cb4b2f02 resolve the missing terms. Will look into the missing subtypes later.

**Comment by perfectly-preserved-pie** [comment]
689f96d6fae2a7c77db416d8f9079ca6b26600b2 fixes the subtype bug. We're close to displaying the whole dataframe now:

`The resulting filtered dataframe has 1950 rows and 1950 markers out of 1965 total rows.`

Still 15 rows missing.

**Comment by perfectly-preserved-pie** [comment]
The problem is back again:
`The resulting filtered dataframe has 2552 rows and 2552 markers out of 3190 total rows.`

~600ish missing markers... why?

**Comment by perfectly-preserved-pie** [comment]
It's the fuckin LaundryFeatures; the filter is too restrictive

**Comment by perfectly-preserved-pie** [comment]
> It's the fuckin LaundryFeatures; the filter is too restrictive

Resolved via 508703eef2eda25850971ae9a5334e9cd19c7657

I expect this issue to crop up again in the future as more LaundryFeatures are added but I'll keep an eye on it.
